fs2-google-pubsub-http/src/main/scala/com/permutive/pubsub/consumer/http/internal/PubsubSubscriber.scala

```diff
@@ -44,7 +47,7 @@ private[http] object PubsubSubscriber {
           subscription = subscription,
           serviceAccountPath = serviceAccountPath,
           config = config,
-          httpClient = httpClient
+          httpClient = Retry[F](httpClientRetryPolicy)(httpClient)
```

Comment:
This change would add `retry` when using the `httpClient` in:
`HttpPubsubReader.scala`
- `def modifyDeadline`
- `def ack`
- `def nack`
- `val read`
`GoogleOAuth.scala`
- `def authenticate`